<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSetup</class>
 <widget class="QWidget" name="CameraSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Image Format</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="shutterSpeedLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="manualExposureLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QRadioButton" name="outputDiscard">
        <property name="text">
         <string>&amp;Discard</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="outputSave">
        <property name="text">
         <string>Save &amp;to..</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="outputDir">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="outputDirButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>ISO</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="imageFormatLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="shoot_mode">
     <item>
      <property name="text">
       <string>Single</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Repeat</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Shoot Mode</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>239</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Manual Exposure</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Shutter Speed</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="imageSettings">
     <property name="text">
      <string>Image Settings...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="isoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QWidget" name="repeated_shots_settings" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Images to capture</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="images_count">
        <property name="specialValueText">
         <string>Infinite</string>
        </property>
        <property name="suffix">
         <string> images</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Shoot Interval</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="shoot_interval">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::HourSection</enum>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="ditherAfterShot">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Dither after each shot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>label_4</zorder>
  <zorder>shutterSpeedLabel</zorder>
  <zorder>label</zorder>
  <zorder>isoLabel</zorder>
  <zorder>imageFormatLabel</zorder>
  <zorder>imageSettings</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>manualExposureLabel</zorder>
  <zorder>verticalSpacer_3</zorder>
  <zorder>label_10</zorder>
  <zorder>shoot_mode</zorder>
  <zorder>groupBox</zorder>
  <zorder></zorder>
  <zorder>repeated_shots_settings</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
